src/components/organisms/TimingDeleteModal/TimingDeleteModal.scss

```
@@ -0,0 +1,14 @@
+@import "scss/constants.scss";
+
+.form {
```

This should probably be encapsulated inside a component class, like
```suggestion
TimingDeleteModal {
  .form {
```

---

done 917110d